exercises/leap/cases_test.go

```diff
@@ -1,7 +1,7 @@
 package leap
 
 // Source: exercism/x-common
-// Commit: be6fa53 leap: Rewrite the test cases and their descriptions
+// x-common version: 1.0.0
```

Comment:
> pulling the canonical-data.json down from Github

The only track I know of that tried to do this is https://github.com/exercism/xcrystal/pull/57, guess that hasn't gotten completed though.

> this would mean not being able to include the commit message in the generated file

How about using https://api.github.com/repos/exercism/x-common/commits?path=exercises/acronym/canonical-data.json  ?

> I assume we wouldn't want to sometimes have the commit message, sometimes not

Yeah seems confusing. I would like to always have it.

But since it seems possible to always have it, looks like it's possible to keep it, yeah?